packages/prisma-client/migrations-cli/cli.ts

```diff
@@ -23,87 +18,92 @@ Commands:
 
 Arguments
   --dev                — Lets the CLI know that it's running in a development environment
+  --force  — Skips the confirmation prompt when running a dangerous command
 `;
 
 const main = async () => {
-  if (args.dev) {
-    dotenv.config();
-  }
-
-  const command = args._[0];
+  try {
```

Comment:
nit: could put this try/catch around main() call so the entire main function has less indentation - makes it easier to read